docs/development/releasing.rst

```diff
@@ -446,6 +496,14 @@ be backported.  For a more detailed set of guidelines on using milestones, see
 Backporting fixes from master
 -----------------------------
 
+.. note::
+
+    The changelog script in ``astropy-procedures`` does not know about
```

Comment:
I removed all mentions of using `add_to_changelog.py` since @bsipocz that the script does not work anyway. Does that mean I should also remove this note, or is this referring to a different "changelog script"?

Reply:
no, this comment is about the scripts in `pr_consistency`, they only check for a PR being backported to a branch, rather than checking for the milestones, too. As the comment says we have two branches: 1) master 2) 1.2.x. The release 1.2.0 will come out of 1.2.x, just as well as 1.2.1. If a PR is merged into master with 1.2.1 it shouldn't be backported until 1.2.0 is out, but the script will complain that it needs to be backported to 1.2.x nevertheless.